src/TConsole.cpp

```diff
@@ -80,7 +80,19 @@ TConsole::TConsole(Host* pH, const QString& name, const ConsoleType type, QWidge
 , mType(type)
 {
     auto quitShortcut = new QShortcut(this);
-    quitShortcut->setKey(Qt::CTRL | Qt::Key_W);
+#if defined(Q_OS_MAC)
+    /*: Shortcut for MacOS to close the current console from within that window
```

Comment:
What do you mean with this? this shortcut is intended to close the profile.

Reply:
Actually - this looks like dead code!

`quitShortcut`'s `QShortCut::activated()` signal is not "connected" to anything AFAICT - and there is not event handler for it within the `TConsole` class.

It is set to have `Qt::WidgetShortcut` so that "The shortcut is active when its parent widget has focus." and in that case a `QEvent::Shortcut` event will occur.

None of our classes have a `QShortcutEvent` handler defined.

:confused:

Reply:
This is academic for **this** PR as I've decided to forget about trying to fix the bodging to get `QShortcut`s to compile cleanly on any Qt version.